docs/support/git-version.md

```diff
@@ -0,0 +1,4 @@
+# Git version requirement
```

Comment:
matching the approach of the node version requirement instead of being included in the FAQs. this also helps because it does not include the specific version in the link, which would need to be updated each time we change our range (and has already resulted in broken links)